projects/tuist/fixtures/app_with_spm_dependencies/Project.swift

```diff
@@ -11,11 +11,17 @@ let project = Project(
             infoPlist: .default,
             sources: "App/**",
             dependencies: [
+                .sdk(name: "libc++.tbd", status: .required),
                 .external(name: "Adjust"),
                 .external(name: "Alamofire"),
                 .external(name: "ComposableArchitecture"),
                 .external(name: "FacebookCore"),
-            ]
+                .external(name: "FirebaseAnalytics"),
+                .external(name: "FirebaseDatabase"),
+            ],
+            settings: .init(base: [
+                "OTHER_LDFLAGS": "-framework \"FirebaseAnalytics\"",
+            ])
```

Comment:
@kwridan when you will fix [this](https://github.com/tuist/tuist/issues/3234) you should remove this setting and the fixture should continue to work 👍